Octokit.Reactive/Clients/ObservableReleasesClient.cs

```diff
@@ -33,7 +33,7 @@ public IObservable<Release> GetAll(string owner, string name)
             Ensure.ArgumentNotNullOrEmptyString(owner, "owner");
             Ensure.ArgumentNotNullOrEmptyString(name, "name");
 
```

Comment:
I think of these as being like contracts. I want them as close to the call-site as possible. Semantically, I think of these arguments being invalid for this method.

Eventually, we'll replace many of these with Fody NullGuard which will throw the exception at the call site.